config/sync/language/da/views.view.user_admin_people.yml

```diff
@@ -19,9 +19,6 @@ display:
           label: Roller
         created:
           label: 'Medlem i'
-          settings:
-            future_format: '@interval'
-            past_format: '@interval'
```

Comment:
Same as others. 

"This is also a thing i keep seeing repeated in config.
I think you mentioned before you wanted to talk about this, but i dont think we ever did."

Reply:
I dunno why it happens - either way, I've just manually removed it now.